packages/workflow/test/handlers.ts

```diff
@@ -66,7 +66,12 @@ const auditEventHandler = rest.post(
   'http://localhost:1050/events/birth/:action',
   (req, res, ctx) => {
     const { action } = req.params
-    const knownActions = ['new-declaration', 'mark-certified', 'mark-issued']
+    const knownActions = [
+      'new-declaration',
+      'mark-certified',
+      'mark-issued',
+      'mark-voided'
```

Comment:
could this refer to a type, using `satisfying` for example? if we update the actions, this would get updates via TSC aswell

Reply:
How would that look like? I am not quite following you

Reply:
I get what you mean now. Hm we can achieve the same using `Pick<AuditAction, ...>`, which I think I've done in another PR